pychargifysimple/web.py

```diff
@@ -3,10 +3,8 @@
 import logging
 
 from mechanicalsoup.browser import LinkNotFoundError
-
```

Comment:
You don't must remove the line break here.

Reply:
I agree that current module import should be separate with by a line break from other module import, to do so and keep isort usage better explicit define current module import as local using dot syntax, in this example:

```python
from .context_manager import login
```